SignInCheckIn/SignInCheckIn/Services/ChildSigninService.cs

```diff
@@ -96,12 +98,14 @@ public ParticipantEventMapDto SigninParticipants(ParticipantEventMapDto particip
 
             //response.Participants.ForEach(p => p.Selected = true);
 
+            /**
```

Comment:
If you could just remove this instead of commenting them out I would prefer instead of having cluttered code.  Unless this is something you aren't sure you want to remove.